api/envoy/config/core/v3/grpc_service.proto

```diff
@@ -43,6 +43,12 @@ message GrpcService {
     string authority = 2
         [(validate.rules).string =
              {min_len: 0 max_bytes: 16384 well_known_regex: HTTP_HEADER_VALUE strict: false}];
+
+    // Indicates the retry policy for re-establishing the gRPC channel
+    // This field is optional. If max interval is not provided, it will be set to ten times the provided base interval.
+    // Currently only supported for xDS gRPC streams.
```

Comment:
Why is this supported only for xDS streams?  Shouldn't this work for any stream?

If this is actually only for xDS streams, then it's not clear to me that it makes sense to put this in `GrpcService`, since that message is used for many things other than xDS streams.

Reply:
The intent was to support this only for xDS streams in this iteration. We can add support for others in further iterations if that is okay.